### PR TITLE
Change 'Me Too!' link after it has been selected - Task 1679

### PR DESCRIPTION
**Pull request by cpeel**
After a user has 'Me Too'd a task, the button goes away. This can lead to confusion why it shows up for users on some pages but not others. It would be better to indicate that they have already "Me Too!'d the task.

While I was in here I did some massaging on the CSS to bring it better inline with HTML5, however it isn't fully compliant.

### CODE REVIEW

File: tasks.php [hunk 1]
```diff
@@ -1081,12 +1088,12 @@ function select_and_list_tasks($sql_condition)
...
     else {
-        echo "<tr bgcolor='#ffffff'><td colspan='7'><center>No tasks found!</center></td></tr>";
+        echo "<tr><td colspan='7'><center>No tasks found!</center></td></tr>";
     }
     echo "</table><br>\n";
     // if 2 tasks or more found, display the number of reported tasks
     if (@mysqli_num_rows($sql_result) > 1) {
-        echo "<p>" . @mysqli_num_rows($sql_result) . " tasks listed.</p>";
+        echo "<small class='task'>" . @mysqli_num_rows($sql_result) . " tasks listed.</small>";
     }
```
Comment (srjfoo) [inline tasks.php]: I'm probably missing something here -- but I'm seeing a switch from a block element to an inline element. Is this already dealt with elsewhere?

**Comment by 70ray** [comment]
Seems ok to me.

File: tasks.php (hunk 1 above)
Comment (wf49670) [reply tasks.php]: I agree it's a change, Sharon. But the text string is inside a div element and that's all that should be needed, technically, I believe. The W3C HTML validator doesn't object to it, at least.

**Comment by wf49670** [comment]
It looks OK to me, if we're not interested (at the moment) in "fixing" the handful of td elements that specify width= directly rather than using CSS or style=.

File: tasks.php (hunk 1 above)
Comment (srjfoo) [reply tasks.php]: Thanks, Walt, that's what I was missing. Yes, a div element is plenty.

**Comment by srjfoo** [comment]
And to me.

**Pull request merged**